rotkehlchen/tests/unit/uniswap/test_calculate_events_balances.py

```diff
@@ -1,130 +0,0 @@
-from typing import TYPE_CHECKING
-from unittest.mock import patch
-
-import pytest
-
-from rotkehlchen.constants import ONE
-from rotkehlchen.tests.utils.ethereum import get_decoded_events_of_transaction
-from rotkehlchen.types import Timestamp, deserialize_evm_tx_hash
-from rotkehlchen.utils.misc import ts_now
-
-from .utils import (
-    TEST_ADDRESS_1,
-    const_lp_1_events_balance,
-    const_lp_2_events_balance,
-    const_lp_3_balance,
-    const_lp_3_events_balance,
-)
-
-if TYPE_CHECKING:
-    from rotkehlchen.api.server import APIServer
-
-
-@pytest.mark.parametrize('ethereum_modules', [['uniswap']])
-def test_no_events_no_balances(rotkehlchen_api_server: 'APIServer') -> None:
-    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
-    uniswap = rotki.chains_aggregator.get_module('uniswap')
-    assert uniswap is not None
-    events_balances = uniswap.get_balances_chain(addresses=[TEST_ADDRESS_1])
-    assert events_balances == {}
-
-
-@pytest.mark.vcr(filter_query_parameters=['apikey'])
```

Comment:
Also we need to make sure that the cassettess are deleted too. Don't leave the files in the cassette repo. At some point we need to figure out if we have files for tests that no longer exist

Reply:
I am removing cassettes too yes. There is a branch there